Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -229,4 +279,51 @@ lemma free_bot (h : A ≠ ⊥) : A.Free (⊥ : SimpleGraph β) := by
 
 end Free
 
+/-!
+#### Induced containment
+
+A graph `H` *inducingly contains* a graph `G` if there is some graph embedding `G ↪ H`. This amounts
+to `H` having an induced subgraph isomorphic to `G`.
+
+We denote "`G` is contained in `H`" by `G ⊴ H` (`\triangle_left_eq`).
+-/
+
+/-- A simple graph `G` is contained in a simple graph `H` if there exists an induced subgraph of `H`
+isomorphic to `G`. This is denoted by `G ⊴ H`. -/
+def IsIndContained (G : SimpleGraph V) (H : SimpleGraph β) : Prop := Nonempty (G ↪g H)
```

Comment:
Do you want a one-field prop-valued structure? I don't have a strong preference

Reply:
And do you want it to be a `class`? in which case I might as well make it an `abbrev`

Reply:
Definitely not a class

Reply:
Then I don't think I want it to be a one-field Prop-valued structure because that would mean it would take `⟨⟨f⟩⟩` to destruct it, instead of `⟨f⟩`